app/build.gradle

```diff
@@ -31,6 +32,12 @@ android {
             GOOGLE_PLACES_API_KEY: GOOGLE_PLACES_API_KEY
         ]
     }
+    tasks.withType(JavaCompile) {
+        // workaround for: https://github.com/google/error-prone/issues/780
+        options.compilerArgs += [ '-Xep:ParameterName:OFF' ]
+        options.compilerArgs += [ '-Xep:MissingOverride:Error']
```

Comment:
This is invalid command line option. Remove it

Reply:
@iamareebjamal Missing ovverride? one or the whole set?

Reply:
Last one

Reply:
@iamareebjamal I have tried every method, but still I am having the same error! kindly chck

Reply:
It's not the same error. You should at least realize that the integration is complete